### ISSUE TITLE
Unknown cleanup policy when visiting topic detail page directly

### ISSUE THREAD

**Issue by IndeedSi** [provectus/kafka-ui#999]
**Describe the bug**
When visiting topic detail page through link /ui/clusters/{cluster}/topics/{topic}, the cleanup policy will always be "UNKNOWN". If you click into the topic from topic list page, correct cleanup policy will be shown.

**Set up**
I compile and run the latest master branch with a Confluent Cloud cluster

**Steps to Reproduce**

1. On topic list page, click into a topic to visit topic detail page.
2. Open a new tab in your browser, copy the same link. 

**Expected behavior**
We should see consistent behavior no matter how we get to topic detail page.

**Screenshots**
When clicking into the topic detail page from list
![image](https://user-images.githubusercontent.com/16617799/138604438-24960815-d9ac-4d79-9b1e-1d2fafff0826.png)

When visit the page in a new tab
![image](https://user-images.githubusercontent.com/16617799/138604453-445664f2-c818-45d7-9082-22349e6ed5b3.png)

**Additional context**
the topic data from topic list
```json
{
    "name": "ad-bidopt-bid",
    "internal": false,
    "partitions": [
        {
            "partition": 0,
            "leader": 10,
            "replicas": [
                {
                    "broker": 10,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 3,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 5,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 2181620,
            "offsetMin": 1943638
        },
        {
            "partition": 1,
            "leader": 20,
            "replicas": [
                {
                    "broker": 20,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 1,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 4672281,
            "offsetMin": 4672264
        },
        {
            "partition": 2,
            "leader": 19,
            "replicas": [
                {
                    "broker": 19,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 20,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 4992288,
            "offsetMin": 4916977
        },
        {
            "partition": 3,
            "leader": 15,
            "replicas": [
                {
                    "broker": 15,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 5,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 10,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 967878,
            "offsetMin": 967803
        },
        {
            "partition": 4,
            "leader": 20,
            "replicas": [
                {
                    "broker": 20,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 1,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 7389784,
            "offsetMin": 7389767
        }
    ],
    "partitionCount": 5,
    "replicationFactor": 3,
    "replicas": 15,
    "inSyncReplicas": 15,
    "segmentSize": 0,
    "segmentCount": 0,
    "underReplicatedPartitions": 0,
    "cleanUpPolicy": "DELETE",
    "id": "2fd0220b-b4a6-408d-8969-6f0c2ebad334"
}
```

topic state from directly accessing topic detail page
```json
{
    "name": "ad-bidopt-bid",
    "internal": false,
    "partitions": [
        {
            "partition": 0,
            "leader": 10,
            "replicas": [
                {
                    "broker": 10,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 3,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 5,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 2181461,
            "offsetMin": 1943638
        },
        {
            "partition": 1,
            "leader": 20,
            "replicas": [
                {
                    "broker": 20,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 1,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 4672118,
            "offsetMin": 4672088
        },
        {
            "partition": 2,
            "leader": 19,
            "replicas": [
                {
                    "broker": 19,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 20,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 4992128,
            "offsetMin": 4916977
        },
        {
            "partition": 3,
            "leader": 15,
            "replicas": [
                {
                    "broker": 15,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 5,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 10,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 967878,
            "offsetMin": 967803
        },
        {
            "partition": 4,
            "leader": 20,
            "replicas": [
                {
                    "broker": 20,
                    "leader": false,
                    "inSync": true
                },
                {
                    "broker": 1,
                    "leader": true,
                    "inSync": true
                },
                {
                    "broker": 12,
                    "leader": true,
                    "inSync": true
                }
            ],
            "offsetMax": 7389619,
            "offsetMin": 7389588
        }
    ],
    "partitionCount": 5,
    "replicationFactor": 3,
    "replicas": 15,
    "inSyncReplicas": 15,
    "segmentSize": 0,
    "segmentCount": 0,
    "underReplicatedPartitions": 0
}
```

**Comment by Haarolean** [comment]
@GneyHabub done, see #1067
![image](https://user-images.githubusercontent.com/1494347/140927789-6528b147-9dfd-4267-8357-35990a012305.png)

**Comment by Haarolean** [comment]
Backend done, waiting for the frontend. Up for grabs.

**Comment by workshur** [comment]
will be fixed by #1045
